BonCodeAJP13/BonCodeAJP13Enums.cs

```diff
@@ -141,6 +141,7 @@ public struct BonCodeAJP13HTTPMethods
         public const byte BONCODEAJP13_MERGE = 0x19;            // MERGE 25 
         public const byte BONCODEAJP13_BASELINE_CONTROL = 0x1A; // BASELINE_CONTROL 26 
         public const byte BONCODEAJP13_MKACTIVITY = 0x1B;       // MKACTIVITY 27 
+        public const byte BONCODEAJP13_SC_M_JKSTORED = 0xFF;       // Any other VERB 255 to Stroe in Attributes
```

Comment:
Even though it's not well-documented on the [official page](https://tomcat.apache.org/connectors-doc/ajp/ajpv13a.html), this functionality has actually been around since at least 2000.

Tracked a bug ([56884](https://bz.apache.org/bugzilla/show_bug.cgi?id=56884)) and peeked at the code documentation for both [Tomcat 10](https://tomcat.apache.org/tomcat-10.0-doc/api/org/apache/coyote/ajp/Constants.html#SC_M_JK_STORED) and [Tomcat 7](https://tomcat.apache.org/tomcat-7.0-doc/api/org/apache/coyote/ajp/Constants.html#SC_M_JK_STORED), as well as the [Tomcat code](https://github.com/apache/tomcat) itself.

In a nutshell: By default, the method looks in the passed-over method slot and saves a few bytes for pre-defined methods. There’s a fallback that tells the protocol to look in the attributes for the method instead. Validated and working from Tomcat 7 to 10.1.

I have been managing about 100 Tomcat apps behind IIS and a lot of them use REST calls with patch, which led me down this rabbit hole.  Looks like the original protocol designers had a solution, just didn’t document it too well.  I am happy to add to the comment but pretty sure it would not be accurate.

Reply:
public const byte BONCODEAJP13_SC_M_JKSTORED = 0xFF;    // Any other VERB 255
//Binary Flag to signal the use of ?stored_method in Attributes
//https://tomcat.apache.org/tomcat-10.0-doc/api/org/apache/coyote/ajp/Constants.html#SC_M_JK_STORED

How about that for an Updated Comment.

Reply:
Thanks for the research. The last comment is good.